src/test/java/uk/gov/justice/digital/delius/transformers/CustodialStatusTransformerTest.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.justice.digital.delius.transformers;
+
+
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.digital.delius.data.api.CustodialStatus;
+import uk.gov.justice.digital.delius.jpa.standard.entity.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CustodialStatusTransformerTest {
+
+    public static final long SENTENCE_ID = 12345678L;
+    public static final String CUSTODIAL_TYPE_CODE = "P";
+    public static final String CUSTODIAL_TYPE_DESCRIPTION = "Post Sentence Supervision";
+    public static final String OFFENCE_DESCRIPTION = "Common assault and battery - 10501";
+    public static final String SENTENCE_DESCRIPTION = "ORA Adult Custody (inc PSS)";
+    public static final LocalDate SENTENCE_DATE = LocalDate.of(2018, 12, 3);
+    public static final LocalDate ACTUAL_RELEASE_DATE = LocalDate.of(2019, 7, 3);
+    public static final LocalDate LICENCE_EXPIRY_DATE = LocalDate.of(2019, 11, 3);
+    public static final Long LENGTH = 11L;
+    public static final String LENGTH_UNIT = "Months";
+
+    @Test
+    public void mapToCustodialStatus() {
+        Disposal disposal = buildDisposal(Collections.singletonList(Release.builder()
+                .actualReleaseDate(ACTUAL_RELEASE_DATE.atTime(13, 0))
+                .build()));
+
+        CustodialStatus custodialStatus = new CustodialStatusTransformer().custodialStatusOf(disposal);
+
+        assertThat(custodialStatus.getSentenceId()).isEqualTo(SENTENCE_ID);
+        assertThat(custodialStatus.getCustodialType().getCode()).isEqualTo(CUSTODIAL_TYPE_CODE);
+        assertThat(custodialStatus.getCustodialType().getDescription()).isEqualTo(CUSTODIAL_TYPE_DESCRIPTION);
+        assertThat(custodialStatus.getSentence().getDescription()).isEqualTo(SENTENCE_DESCRIPTION);
+        assertThat(custodialStatus.getMainOffence().getDescription()).isEqualTo(OFFENCE_DESCRIPTION);
+        assertThat(custodialStatus.getSentenceDate()).isEqualTo(SENTENCE_DATE);
+        assertThat(custodialStatus.getActualReleaseDate()).isEqualTo(ACTUAL_RELEASE_DATE);
+        assertThat(custodialStatus.getLicenceExpiryDate()).isEqualTo(LICENCE_EXPIRY_DATE);
+        assertThat(custodialStatus.getLength()).isEqualTo(LENGTH);
+        assertThat(custodialStatus.getLengthUnit()).isEqualTo(LENGTH_UNIT);
+    }
+
+    @Test
+    public void givenMultipleReleases_thenActualReleaseDateIsTheLatest() {
+        Disposal disposal = buildDisposal(Arrays.asList(
```

Comment:
I've had to make an assumption here about what the correct behaviour is if there are multiple releases associated with a Disposal. I went for the simplest thing that seemed sensible but did consider whether it would be better to return the latest release that is not in the future? As it stands if a release is in the future it will nonetheless be returned but I'm not sure if a release can be in the future.

Reply:
I very much doubt it, I suspect UI constrains it. If you are concerned we could always check database to see if there are any